docs/MVS_Outputs.rst

```diff
@@ -155,6 +155,47 @@ To measure this, we propose to compare the energy flows in between the sectors t
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Self consumption (SC)
+###############################
+
+The self consumption describes the fraction of all renewable energy that is consumed by the system itself.
+
+.. math::
+        SC &=\frac{\sum_{i,j} {E_{generation} (i,j) \cdot w_i} - E_{gridfeedin}(i,j) \cdot w_i}{\sum_{i,j} {E_{generation} (i,j) \cdot w_i}}
+
+        \text{with~} &i \epsilon \text{[PV,Geothermal,…]}
+
+        &j \epsilon \text{[generation assets 1,2,…]}
```

Comment:
This results in following equation:
![grafik](https://user-images.githubusercontent.com/44204527/96568780-99525100-12c8-11eb-8e66-108f6ccdf597.png)
I am wondering why you have `(i,j)` for your energy flows. Isnt `i` only sufficient? 

If you got it from the degree of sector-coupling: Only there we look at the conversion of energy carriers, in a specific asset from `i` to `j`. This does not happen in your KPI.

If this is what happened, then you can delete the `i,j` from all of your equations.

Reply:
Maybe you can even get rid of the index explaination at the foot of the equation. I am not sure if it applies to your case...?

Reply:
I am not sure to be honest how the iteration works in mvs. I thought I would iterate over all sorce types and then over all assets within this source type, as in sector coupling. After your comment I fixed it to only i \element generation asset. 

I think you know better here how the flows are sumed up in mvs so just let me know how the convention is.